binding.go

```diff
@@ -37,6 +48,11 @@ func JSON(model interface{}) flamego.Handler {
 			}
 		}
 		validateAndMap(c, validate, obj, errs)
+
+		errs = c.Value(reflect.TypeOf(errs)).Interface().(Errors)
+		if len(errs) > 0 && option.ErrorHandler != nil {
+			_, _ = c.Invoke(option.ErrorHandler)
```

Comment:
Should we panic on error here? Swallow error would cause confusion

Reply:
Fixed.